back-end/routes/api.php

```diff
@@ -41,6 +41,8 @@
     });
 });
 
+
+Route::get('/leave/leave-status',  [LeaveController::class, 'getLeaveStatus'])->middleware('auth');
```

Comment:
```suggestion
Route::get('/leave/leave-requests',  [LeaveController::class, 'getLeaveStatus'])->middleware('auth');
```

Van de controller functie lijkt dit mij een endpoint waar alle leave requests worden gereturned. Dan lijkt dit mij een beter endpoint